src/app/pages/administration/svaps/svaps.component.spec.js

```diff
@@ -0,0 +1,333 @@
+(() => {
+    'use strict';
+
+    fdescribe('the SVAP Administration page', () => {
```

Comment:
```suggestion
    describe('the SVAP Administration page', () => {
```

All tests ought to be run when PRs are submitted